api/api/admin/media_report.py

```diff
@@ -106,6 +193,165 @@ def pending_reports_links(self, obj):
 
         return mark_safe(", ".join(data))
 
+    def changelist_view(self, request, extra_context=None):
+        extra_context = extra_context or {}
+
+        extra_context["media_type"] = self.media_type
+
+        valid_locks = self.lock_manager.prune()
+        locked_media = list(
+            int(item.replace(f"{self.media_type}:", ""))
+            for lock_set in valid_locks.values()
+            for item in lock_set
+            if self.media_type in item
```

Comment:
This doesn't filter out the items locked by the requester. I could see that being intentional, but it conflicts the note at the top which says "moderator by another user". This change fixes it for me locally:

```suggestion
            for moderator, lock_set in valid_locks.items()
            for item in lock_set
            if moderator != request.user.get_username() and self.media_type in item
```

Alternatively, if you feel that it should highlight works in moderation by the requester as well, then we should change the text of the note to reflect that (remove the "another user" aspect).

Reply:
No, you're right. Fixed in cde29dc.